src/neomaril_codex/base.py

```diff
@@ -5,7 +5,7 @@
 from typing import Optional
```

Comment:
For both files I think it is a good idea to use the black formatter and ruff

Reply:
https://github.com/datarisk-io/mlops-neomaril-codex/pull/63/commits/86d570d11eecefc34b1c21678663d7e39de849b3